config.mak.uname

```diff
@@ -718,19 +710,13 @@ else
 		HAVE_LIBCHARSET_H = YesPlease
 		USE_GETTEXT_SCHEME = fallthrough
 		USE_LIBPCRE = YesPlease
-		USE_NED_ALLOCATOR = YesPlease
 		ifeq (/mingw64,$(subst 32,64,$(prefix)))
 			# Move system config into top-level /etc/
 			ETC_GITCONFIG = ../etc/gitconfig
 			ETC_GITATTRIBUTES = ../etc/gitattributes
 		endif
-	else
-		COMPAT_CFLAGS += -D__USE_MINGW_ANSI_STDIO
-		NO_CURL = YesPlease
-		NO_PYTHON = YesPlease
 	endif
 endif
-endif
```

Comment:
The commit message "Remove MSys Support" is way too terse. Please do study https://github.blog/2022-06-30-write-better-commits-build-better-projects/ and apply its guidance.

As I've mentioned before, compare this too-short oneliner with the commit message of https://github.com/gitgitgadget/git/commit/df5218b4c30b2fa1ba0d1ada4166ad85bc83be26. The latter is okay by Git's standards, the former is absolutely not up to muster.